main.go

```diff
@@ -1,20 +1,82 @@
 package main
 
 import (
-	"github.com/dmacvicar/terraform-provider-libvirt/libvirt"
-	"github.com/hashicorp/terraform/plugin"
+	"flag"
+	"fmt"
+	"log"
 	"math/rand"
+	"os"
 	"time"
+
+	"github.com/dmacvicar/terraform-provider-libvirt/libvirt"
+	"github.com/hashicorp/terraform/plugin"
+	libvirtgo "github.com/libvirt/libvirt-go"
```

Comment:
`nitpick` is better to stay coherent with the other names: 
https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/libvirt/domain.go#L13

Reply:
`libvirt`

Reply:
A yeah   nice